dumb.scss

```diff
@@ -0,0 +1 @@
+.dumb { .scss { .is { .dumb { .i { .dont { .know { .how { .else { .to { .write { .bad { .scss { font-size: 700em; } } } } } } } } } } } } }
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 3<br>Selector should have depth of applicability no greater than 2, but was 4<br>Selector should have depth of applicability no greater than 2, but was 5<br>Selector should have depth of applicability no greater than 2, but was 6<br>Selector should have depth of applicability no greater than 2, but was 7<br>Selector should have depth of applicability no greater than 2, but was 8<br>Selector should have depth of applicability no greater than 2, but was 9<br>Selector should have depth of applicability no greater than 2, but was 10<br>Selector should have depth of applicability no greater than 2, but was 11<br>Selector should have depth of applicability no greater than 2, but was 12<br>Selector should have depth of applicability no greater than 2, but was 13